### PR TITLE
Added styling for github logo #24

### PR DESCRIPTION
**Pull request by OsafAliSayed**
#22

### CODE REVIEW

**Comment by lokendarjangid** [comment]
@OsafAliSayed please mention the issue number here with # so that it will be automatically closed

File: static/style.css [hunk 1]
```diff
@@ -89,6 +89,16 @@ body {
     background:linear-gradient(rgba(0, 0, 0, 0.294),rgba(255, 255, 255, 0.437)) ;
 }
 
+.source-heading {
```
Comment (lokendarjangid) [inline static/style.css:92]: make github icon little bit left side like not touching the right side of window

**Comment by OsafAliSayed** [comment]
Done please review and let me know if further changes are required.

**Comment by lokendarjangid** [comment]
mention the issue number please so it will automatically close.

**Review by lokendarjangid — COMMENTED** [review]
all things all perfect

**Review by lokendarjangid — APPROVED** [review]

**Pull request merged**